stories/About/Developer Guide.mdx

```diff
@@ -0,0 +1,236 @@
+import { Meta } from "@storybook/blocks";
+import Image from "next/image";
+
+import supabase from '../assets/supabase_gridIron.png';
+import server from '../assets/supabase_server.png';
+import client from '../assets/client_component.png';
+  
+<Meta title="About/Developer Guide" />
+
+<div className="sb-container">
+  <div className='sb-section-title'>
```

Comment:
Inconsistent usage of double and single quotes. Work with @ryandotfurrer on the coding standards for this project.